packages/main/src/TableHeaderCellActionAI.ts

```diff
@@ -0,0 +1,39 @@
+import { customElement, i18n } from "@ui5/webcomponents-base/dist/decorators.js";
+import TableHeaderCellActionBase from "./TableHeaderCellActionBase.js";
+import { TABLE_GENERATED_BY_AI } from "./generated/i18n/i18n-defaults.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import "@ui5/webcomponents-icons/dist/ai.js";
+
+/**
+ * @class
+ *
+ * ### Overview
+ *
+ * The `ui5-table-header-cell-action-ai` component defines a dedicated AI action for the table column.
+ *
+ * ### ES6 Module Import
+ *
+ * `import "@ui5/webcomponents/dist/TableHeaderCellActionAI.js";`
+ *
+ * @constructor
+ * @extends TableHeaderCellActionBase
+ * @since 2.7.0
```

Comment:
I guess the version should be 2.8 now? Or is this going to be part of 2.7 as well?